src/kintone/client.ts

```diff
@@ -63,33 +69,11 @@ const buildHttpsAgent = (options: {
     return new https.Agent({ ...clientAuth });
   }
 
-  const { protocol, hostname, port, username, password } = new URL(
-    options.proxy,
-  );
-  const proxyOptions: httpsProxyAgent.HttpsProxyAgentOptions = {
-    protocol: protocol,
-    host: hostname,
-    port: port,
+  const proxyOptions: HttpsProxyAgentOptions<ProxyAgentOptions> = {
     ...clientAuth,
   };
 
-  if (username.length > 0 && password.length > 0) {
```

Comment:
NOTE: Proxy authentication is now controlled by https-proxy-agent
Ref: https://github.com/TooTallNate/proxy-agents/blob/main/packages/https-proxy-agent/src/index.ts#L112-L119

Reply:
Great!
It makes our code simpler 😎